returnn/datasets/basic.py

```diff
@@ -893,20 +877,6 @@ def serialize_data(self, key, data):
       return vocab.labels[data]
     return vocab.get_seq_labels(data)
 
-  def calculate_priori(self, target="classes"):
-    """
-    :param str target:
-    :rtype: numpy.ndarray
-    """
-    priori = numpy.zeros((self.num_outputs[target][0],), dtype=numpy.float32)
-    i = 0
-    while self.is_less_than_num_seqs(i):
-      self.load_seqs(i, i + 1)
-      for t in self.get_targets(target, i):
-        priori[t] += 1
-      i += 1
-    return numpy.array(priori / self.get_num_timesteps(), dtype=numpy.float32)
```

Comment:
This is somewhat unrelated to this PR. But I guess also ok.

Reply:
Ah yes, `update_weights` is extra as well. I forgot to exclude it from the commit.